src/templates/rpc/Dockerfile_rpc

```diff
@@ -11,7 +11,7 @@ WORKDIR /root/warnet
 
 # Copy the source directory contents into the container
 COPY . /root/warnet
-RUN pip install .
+RUN pip install -e .
```

Comment:
I don't think we want this in the RPC Dockerfile here, can't we just restore the `MANIFEST.in` file and use that?

It seems useful to test that proper install works...

Reply:
will delete this